services/web/project/templates/base.html

```diff
@@ -44,10 +44,14 @@
     </div>
 
 
+    <!-- JQuery 3.6.0 minified link -->
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
     <!--Bootstrap javascript link-->
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
         integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
         crossorigin="anonymous"></script>
+    <!-- question.html javascript link -->
+    <script type="text/javascript" src="{{ url_for('views.staticfile', folder='js', filename='question.js') }}"></script>
 </body>
```

Comment:
Script declaration should be in the head and not the body.  This is most likely why you are running into issues. If you look at the base.html and question.html in #92 you will see how it should be set up.

Reply:
Not necessarily;
>The SCRIPT element places a script within a document. This element may appear any number of times in the HEAD or BODY of an HTML document.
From [here](https://www.w3.org/TR/html4/interact/scripts.html#h-18.2.1)

What issues are you referring to? Code works as expected. The issues were with the Jinja block in the question.html file.